README.md

```diff
@@ -1,3 +1,4 @@
+<!-- party! -->
 <!--TODO(bentheelder): fill this in much more thoroughly-->
```

Comment:
want to make it a real PR and remove this todo? :joy:

Reply:
It is done